src/api/login.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/json9512/mediumclone-backendwithgo/src/dbtool"
+)
+
+// Login validates the user and distributes the tokens
+func Login(p *dbtool.Pool) gin.HandlerFunc {
+	handler := func(c *gin.Context) {
```

Comment:
이런식으로 안되려나? 이게 조금 더 가독성있을 것 같아서

```suggestion
func Login(p *dbtool.Pool) gin.HandlerFunc {
	return func(c *gin.Context) {
```

Reply:
```go
// HandlerFunc defines the handler used by gin middleware as return value.
type HandlerFunc func(*Context)

// GET is a shortcut for router.Handle("GET", path, handle).
func (group *RouterGroup) GET(relativePath string, handlers ...HandlerFunc) IRoutes {
	return group.handle(http.MethodGet, relativePath, handlers)
}
```
찾아보니까 `func(*Context)` 만 리턴하면 되는 거라 제안 한 방법으로도 되네용.